mlir/include/mlir/Interfaces/TilingInterface.td

```diff
@@ -74,6 +74,25 @@ def TilingInterface : OpInterface<"TilingInterface"> {
           return {};
         }]
       >,
+      InterfaceMethod<
+        /*desc=*/[{
+          Method to return iterator domain position computed by the
```

Comment:
I think this isnt as clear. Please see the description for `getResultTilePosition` maybe something closer to that (or improve both if you have a better description).